exporter/prometheusexporter/config.go

```diff
@@ -40,4 +40,8 @@ type Config struct {
 
 	// MetricExpiration defines how long metrics are kept without updates
 	MetricExpiration time.Duration `mapstructure:"metric_expiration"`
+
+	// ResourceAttributesAsTags, if sMet to true, will use the exporterhelper feature to
+	// transform all resource attributes into metric labels.
+	ResourceAttributesAsTags bool `mapstructure:"resource_attributes_as_tags"`
```

Comment:
In contrib where we use this, we have config like:

```
	exporterhelper.ResourceToTelemetrySettings `mapstructure:"resource_to_telemetry_conversion"`
```

so the config will be:

```
   resource_to_telemetry_conversion:
      enabled: true
```

If you don't like that name maybe embed that settings but use the name `resource_attributes_as_tags`? What do you think?

Reply:
ah cool, it should be consistent with existing implementations so happy to switch it up to:

``` yaml
        resource_to_telemetry_conversion:
            enabled: true
 ```